src/raftstore/store/peer.rs

```diff
@@ -1953,7 +1961,7 @@ impl Peer {
         let last_ready_read_count = self.raft_group.raft.ready_read_count();
 
         let id = self.pending_reads.next_id();
-        let ctx = id.to_ne_bytes();
+        let ctx = (((id as u128) << 64) | self.peer.get_id() as u128).to_ne_bytes();
```

Comment:
Does the `ctx` cross tikv instances? If so, please use `to_le_bytes` or `to_be_bytes` instead, because different instance may have different endian.